v2/support/secure_settings_sample_app/README.md

```diff
@@ -0,0 +1,20 @@
+# Secure Settings Example App
+
+This sample app shows how to add Secure Settings to an app. "Secure Settings" is a neat feature that allows your App to pass a secure setting via the proxy that is invisible to normal agents.
+
+In particular, this App uses a secure setting to store a token for https://thecatapi.com/ which is required for making api requests.  To get such a key, you will need to sign up for an account, and then grab your api key from the signup email.
+
+On installation the account admin should paste this key into the "catering_token" field.
+
+The app will log to the browser console a single response from thecatapi, which is a GET request to /images/search per https://documenter.getpostman.com/view/4016432/RWToRJCq .
```

Comment:
Could you update this to match the new functionality?

Reply:
Good point, done.